### ISSUE TITLE
JWT token scalar?

### ISSUE THREAD

**Issue by carlocorradini** [Urigo/graphql-scalars#523]
What do you think about a JWT scalar type?

**Comment by TheGrimSilence** [comment]
Is it possible to get more info on this? Preferred implementation, why you feel it's necessary, how it works in the context of GraphQL schemas?

**Comment by carlocorradini** [comment]
For example in a Sign In mutation. When the User is authenticated a JWT token is returned to the client.

Or in combination with [TypeGaphQL](https://github.com/MichalLytek/type-graphql) where in the Mutation I assign the return value to be a GraphQLJsonWebToken (or GraphQLJWT) and not a generic string.

```typescript
@Mutation(() => String)
  signIn(
    @Arg('username', () => GraphQLNonEmptyString) username: string,
    @Arg('password', () => GraphQLNonEmptyString) password: string,
 ): Promise<string> {
    return this.userRepository.signIn(username, password);
}
```

**Comment by ardatan** [comment]
Available in v1.8.0! Thanks @carlocorradini <3
